stdlib/ballerina-builtin/src/main/ballerina/runtime/Package.md

```diff
@@ -0,0 +1,100 @@
+## Package overview
+
+This package includes functions to interact with the runtime, the invocation context and to manage errors.
+
+### Invocation Context
+
+The Invocation Context is a data holder which is created per request, and preserved for a single request-response flow. 
+The Invocation Context comprises of a unique id, UserPrincipal instance which includes user details and an AuthContext 
+instance which has the authentication related details if available.
+
+### Errors
+
+Error types NullReferenceException and IllegalStateException, which wraps the error type 
+defined in ballerina/builtin package, are included in the runtime package. Furthermore, there are utility methods to 
+retrieve the current call stack and the particular call stack frame for an error. 
+
+Additionally, the runtime package also contains util methods to halt a thread (sleep) for a given period of time and 
+to lookup properties from the runtime context.
```

Comment:
Suggestions:
1. Have 'util' within single quotes to have it in code format because it is the name of a method.
2. Change "lookup" to "look up" because it is a verb in this context.

Reply:
The util here should not be highlighted as code, since what its supposed to say is 'utility methods'. I will change util to utility to make that clearer.